packages/react-core/src/demos/DataListDemo.md

```diff
@@ -418,3 +426,340 @@ class ExpandableDataList extends React.Component {
   }
 }
 ```
+
+### Basic
+
+```js isFullscreen
```

Comment:
WE are in t he process of converting all our Demo's to typescript.  Can we convert this one as well.